dist/s_all

```diff
@@ -22,8 +22,32 @@ while len(pids):
 ' $$
 }
 
+CLRL=$'\e[K'     # Clear to the end of line.
+CLRS=$'\e[0J'    # Clear to the bottom of screen.
+BOL=$'\r'        # Move cursor to the beginning of line.
+# `stty` unfortunately uses stdin (not stdout) to locate the tty so have to check both.
+[[ -t 1 && -t 0 ]] && { ISTTY=1; CLRIFTTY="$CLRS"; }
+
+status() {
+    # Single line version: ensures that the status soesn't span for more than one line.
+    # [[ -n "$ISTTY" ]] && echo -n "${1:0:`stty size | sed -n 's/^[0-9]* //p'`}$CLRL$BOL"
+
+    # Multiline version: counts the number of lines and moves the cursor up accordingly.
```

Comment:
If we don't need the above, we can remove this.

Reply:
Done